### PR TITLE
Sitecore: New article: Connect to Azure SQL

### PR DESCRIPTION
**Pull request by DonNateR**
We still need to define the rules for linking to Azure documentation.

### CODE REVIEW

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-a40fe548e7/how-to/). :bow:

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-a81edc2e2a/how-to/). :bow:

**Comment by kmsholcomb** [comment]
Hi @DonNateR. Edits are attached. Please let me know if you have any questions. Thanks!

[Connect to Azure SQL_editsKH.docx](https://github.com/rackerlabs/rackspace-how-to/files/1056090/Connect.to.Azure.SQL_editsKH.docx)
[Connect to Azure SQL_editsKH.pdf](https://github.com/rackerlabs/rackspace-how-to/files/1056091/Connect.to.Azure.SQL_editsKH.pdf)

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-d53087938d/how-to/).

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-39e2e1786e/how-to/)

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-d44b2445c0/how-to/).

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-f552b07184/how-to/).

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-26eeaab2a2/how-to/). :bow:

File: content/managed-operations/connect-to-azure-sql-database.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+type: article
+created_date: '2017-05-31'
+created_by: Juan Garza
+last_modified_date: '2017-05-31'
+last_modified_by: Nate Archer
+product: Managed Operations
+product_url: managed-operations
+---
+
+When a change to an application works in a development environment but does not work in production, you might need to make that same change in your database. Microsoft Azure SQL Database uses the same tools as a SQL Server, but with the capability of the Microsoft cloud.
```
Comment (catlook) [inline content/managed-operations/connect-to-azure-sql-database.md]: ... SQL Server but with ... [ don't think you need the comma here ]

File: content/managed-operations/connect-to-azure-sql-database.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+This article describes how to connect to an Azure SQL Database.
+
+### Prerequisite
+
+- Your login credentials for the Azure portal. For information about how to log in to the Azure portal, see [Sitecore Cloud portals and account management](/how-to/sitecore-cloud-portals-and-account-management/).
+
+### Install the latest version of SQL Server Management Studio
+
+When you are working with SQL Database, you should always use the latest version of SQL Server Management Studio (SSMS). The latest version of SSMS is optimized to work with Azure and SQL Database, and works for all supported versions of SQL Server. To help you stay up-to-date, the latest version prompts you when there is a new version available to download.
```
Comment (catlook) [inline content/managed-operations/connect-to-azure-sql-database.md]: ..  SQL Database and works ...

File: content/managed-operations/connect-to-azure-sql-database.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+### Install the latest version of SQL Server Management Studio
+
+When you are working with SQL Database, you should always use the latest version of SQL Server Management Studio (SSMS). The latest version of SSMS is optimized to work with Azure and SQL Database, and works for all supported versions of SQL Server. To help you stay up-to-date, the latest version prompts you when there is a new version available to download. 
+
+To download and install the latest version of SMSS, see [Download SQL Server Management Studio](https://docs.microsoft.com/en-us/sql/ssms/download-sql-server-management-studio-ssms).
+
+### Create a server-level firewall rule
+
+By default, an Azure SQL Database firewall prevents external connectivity to your logical server and its databases. To enable you to connect to your server, you need to create a firewall rule for the IP address of the computer from which you connect.
```
Comment (catlook) [inline content/managed-operations/connect-to-azure-sql-database.md]: ...  which you connect by using the following instructions:

**Comment by DonNateR** [comment]
Should be good to merge now.

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-64ffffb6b6/how-to/). :bow:

**Pull request merged**